nsm-test/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "nsm-test"
+version = "0.1.0"
+authors = ["Andrei Trandafir <aatrand@amazon.com>"]
```

Comment:
I think we should list this as "The AWS Nitro Enclaves team" and point to our mailing list instead. (Should do this retroactively as well).